src/components/ModuleFields/errors.vue

```diff
@@ -6,7 +6,7 @@
       force-show
       class="m-0"
     >
-      {{ error.message }}
+      {{ $t(`field:${error.message}`) }}
```

Comment:
Problem with this is we have custom validation and therefore also custom error.messsages.
So you should only do $t(error.message) here. That either translates it or keeps it as is if it doesn't have a translation